Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -310,6 +310,24 @@ enum TrackingEvent: String, Trackable {
     case screenOnboarding3                          = "screen_onboarding_3"
     case onboardingSkipped                          = "user_skip_onboarding"
 
+    // Share link
+    case screenAddKeyAsOwnerShareInfo               = "screen_add_key_as_owner_share_info"
+    case userSkipShareLink                          = "user_skip_share_link"
+    case screenAddOwnerShareLink                    = "screen_add_owner_share_link"
+    case userAddOwnerShareLink                      = "user_add_owner_share_link"
+
+    // Handle link
+    case screenOwnerFromLink                        = "screen_owner_from_link"
+    case userRejectOwnerFromLink                    = "user_reject_owner_from_link"
```

Comment:
you know, maybe at some point we should do a refactor/rename and rename all these cases to look exactly as their values - that way it would be easier to search in the codebase

Reply:
That would be so great! What direction?

`case userRejectOwnerFromLink                    = "userRejectOwnerFromLink"`
or
`case user_reject_owner_from_link                    = "user_reject_owner_from_link"`